### ISSUE TITLE
[docker] optional login for registries

### ISSUE THREAD

**Issue by gotson** [jreleaser/jreleaser#1147]
**Is your feature request related to a problem? Please describe.**
I want to use JReleaser along with Github Actions, and https://github.com/docker/login-action

Since the action is going to perform the login, i don't want JReleaser to login again, and i don't want to supply JReleaser with the login credentials.

**Describe the solution you'd like**
A way to disable login for docker registries.

**Describe alternatives you've considered**
Changing my Github workflow so that i can pass the Docker credentials to JReleaser

**Comment by gotson** [comment]
Basically i'd like my config to look like:

```kotlin
      registries {
        create("docker.io")
        create("ghcr.io")
      }
```

**Comment by gotson** [comment]
raised from https://github.com/jreleaser/jreleaser/discussions/1138#discussioncomment-4598453

**Comment by aalmiray** [comment]
Released in v1.5.0 -> https://github.com/jreleaser/jreleaser/releases/tag/v1.5.0
